lib/utils/sdp-utils.js

```diff
@@ -12,6 +12,30 @@ const mergeSdpMedia = (sdp1, sdp2) => {
   return sdpTransform.write(parsedSdp1);
 };
 
+const isOpusFirst = (sdp) => {
+  const parsedSdp = sdpTransform.parse(sdp);
+  const idx = parsedSdp.media[0].rtp.findIndex((entry) => entry.codec === 'opus');
+  return idx === 0;
+};
+
+const opusFirst = (sdp) => {
+  const parsedSdp = sdpTransform.parse(sdp);
+  // Find the index of the OPUS codec
+  const opusIndex = parsedSdp.media[0].rtp.findIndex((entry) => entry.codec === 'opus');
+
+  // Move OPUS codec to the beginning
+  if (opusIndex !== -1) {
```

Comment:
why are we comparing to -1 (not found)?  What if opusIndex is 0?  Then we should just leave it alone.  What if opusIndex is 3?  Then we should be moving it, but it seems we would be leaving it there

Reply:
right, we just need to move if the index of opus > 0. 
Fixed